soc/arm/renesas_smartbond/Kconfig.defconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2022 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_FAMILY_SMARTBOND
+
+source "soc/arm/renesas_smartbond/*/Kconfig.defconfig.series"
+
+#config SYS_CLOCK_HW_CYCLES_PER_SEC
+#	default 32000000
```

Comment:
This should be picked from devicetree as below: https://github.com/zephyrproject-rtos/zephyr/blob/c699ae5850297c824f8a7ca2a4f06174a9782fd3/soc/arm/gigadevice/gd32f4xx/Kconfig.defconfig.gd32f470#L9-L10

BTW, is this the CPU core frequency or just crystal value?